### DISCUSSION TITLE
/sleep like /sit

### DISCUSSION THREAD

**Discussion by Kryniowesegryderiusz** [Zrips/CMI#3282] [Feature Requests]
**Description of feature/improvement request:**
It would be great if it could be possible to lay on the ground like its possible to /sit now. It would look like player in bed, but on the ground

**Comment by Kryniowesegryderiusz** [comment]
bump? 😂

**Comment by CloudeLecaw** [comment]
Wonderful idea

https://github.com/Zrips/CMI/discussions/6708


pls also add /swim ^^
